app/src/main/res/layout/content_main.xml

```diff
@@ -56,18 +61,27 @@
         android:id="@+id/txtActivity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginTop="92dp"
         android:ems="10"
+        android:hint="@string/activity_hint"
         android:inputType="textPersonName"
-        android:text="Activity..."
-        tools:layout_editor_absoluteX="99dp"
-        tools:layout_editor_absoluteY="190dp" />
+        android:text="@string/activity"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView" />
 
     <Button
         android:id="@+id/button2"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Add Activity"
-        tools:layout_editor_absoluteX="59dp"
-        tools:layout_editor_absoluteY="375dp" />
+        android:layout_marginStart="5dp"
+        android:layout_marginLeft="5dp"
+        android:layout_marginTop="61dp"
+        android:layout_marginEnd="6dp"
+        android:layout_marginRight="6dp"
+        android:text="@string/add_activity"

```

Comment:
Created string in strings.xml and replaced text with @string. This removed this warning in android studio about using string values.